dash-pipeline/bmv2/dash_routeaction.p4

```diff
@@ -0,0 +1,205 @@
+#ifndef _ROUTEACTION_P4_
+#define _ROUTEACTION_P4_
+
+#include "dash_headers.p4"
+
+/*
+ * Packet transformation actions
+ */
+control action_staticencap(inout headers_t hdr, inout metadata_t meta)
+{
+    action drop() {
+        meta.dropped = true;
+    }
+
+    apply {
+        if (meta.encap_data.encap_type == dash_encapsulation_t.VXLAN) {
+            vxlan_encap(hdr,
```

Comment:
we can move the vxlan_encap and nvgre_encao into "routing_actions/action_staticencap.p4".

Reply:
vxlan_encap/vxlan_decap are still used in dash_inbound.p4 and dash_pipeline.p4. Will do it later.